_rules/SC4-1-2-role-has-required-states-and-properties.md

```diff
@@ -0,0 +1,75 @@
+---
+name: Role has required states and properties
+description: |
+  Elements that has a semantic role must also specify all required states and properties
+  
+success_criterion:
+- 4.1.2 # Name, Role, Value
+
+test_aspects:
+- DOM Tree
+- CSS Styling
+
+authors:
+- Anne Thyme Nørregaard
+---
+
+## Test procedure
+
+### Applicability
+
+Any HTML or SVG element that is [exposed to assistive technologies](#exposed-to-assistive-technologies) and has a [semantic role](#semantic-role). 
```

Comment:
Some more thoughts. For the aria-valid-attribute-value rule I'm going to allow elements not exposed to assistive technologies, otherwise elements with aria-hidden="true" would be inapplicable instead of passed. I think we should be consistent between these rules about the exposed to assistive technologies part.

Reply:
Per the HTML AAM, the selectedness of `<option>` is mapped to `aria-selected` (https://www.w3.org/TR/html-aam/#details-id-103) so `aria-selected` is implicitly set.

Reply:
Unless there's a case of an element that is implicitly mapped to an ARIA role, but doesn't define native attributes that are mapped to all of the required ARIA states and properties, I'd also agree that we can leave out elements with implicit roles.

Reply:
@WilcoFiers: I agree that we should have consistency between rules, but I don't see it as a good solution to create consistency between these two rules and sacrificing the consistency between them and the rest that handle things that are (primarily) important for screen reader users, e.g. all of rules that checks for accessible names.